packages/grafana-data/src/field/fieldState.ts

```diff
@@ -47,9 +47,9 @@ export function getFieldDisplayName(field: Field, frame?: DataFrame, allFrames?:
 }
 
 /**
- * Get an appropriate display name. If the 'title' is set, use that
+ * Get an appropriate display name. If the 'displayName' field config is set, use that
  */
-function calculateFieldDisplayName(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
+export function calculateFieldDisplayName(field: Field, frame?: DataFrame, allFrames?: DataFrame[]): string {
```

Comment:
why export this and not use getFieldDisplayName (the exported function), this is an internal function

Reply:
Change to `getFieldDisplayName` to get the benefit of using the cached field name.

Reply:
fixed in 655357e